.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,30 @@
 Fixes #
 
-#### Changes proposed in this Pull Request:
+### Changes proposed in this Pull Request
```

Comment:
Can we remove this from all templates? We can use descriptive titles and use those as the change log entry.